vm/src/pyobject.rs

```diff
@@ -1574,25 +1575,25 @@ impl PyObject {
 }
 
 pub trait PyValue: fmt::Debug + Sized + 'static {
-    fn required_type(ctx: &PyContext) -> PyObjectRef;
+    fn class(vm: &mut VirtualMachine) -> PyObjectRef;
 
-    fn into_ref(self, ctx: &PyContext) -> PyRef<Self> {
+    fn into_ref(self, vm: &mut VirtualMachine) -> PyRef<Self> {
```

Comment:
This change is unfortunate. I was hoping to able to use this in PyContext without a vm.

Reply:
At some point pyContext and vm should probably be merged into one. It does not make sense to have both I believe.